### DISCUSSION TITLE
Smaller screen as iphone displays

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#72] [General]
On smaller screen, some part of the page need to be scrolled horizontally to be displayed which is not so UX friendly as following : 

![Screenshot 2023-05-24 at 19 34 56](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/fbbf6972-9c41-4d79-be65-336a00d56681)

**Comment by TJ-BitCanna** [comment]
Thanks for noticing! We are also looking into responsiveness of the webwallet and a new design overall which is in line with our mobile app which we are developing. 

I'll get back on this after internal discussion what we'll do time-wise regarding design.

**Comment by TJ-BitCanna** [comment]
We'll be looking into how the design looks on various screen sizes, although using it on mobile devices (iPhone, iPad, etc) is not possible due to missing the ability to install Keplr as an extension on these devices.

Usage will be limited to laptop(-like) and desktop devices anyway.

I did some testing for this and also on smaller screen sizes there is no scrollbar. However, the formatting needs a look, but that is where we will be working on in the coming period.
